sdk/cosmos/azure-cosmos/samples/document_management.py

```diff
@@ -171,16 +171,61 @@ def patch_item(container, doc_id):
                                                  response["service_addition"]))
 
 
+def execute_item_batch(database):
+    print('\n1.10 Executing Batch Item operations\n')
+    container = database.create_container_if_not_exists(id="batch_container",
+                                                        partition_key=PartitionKey(path='/account_number'))
+    # We create three items to use for the sample.
+    container.create_item(get_sales_order("read_item"))
+    container.create_item(get_sales_order("delete_item"))
+    container.create_item(get_sales_order("replace_item"))
+
+    # We create our batch operations
+    create_item_operation = ("create", (get_sales_order("create_item"),))
+    upsert_item_operation = ("upsert", (get_sales_order("upsert_item"),))
+    read_item_operation = ("read", ("read_item",))
+    delete_item_operation = ("delete", ("delete_item",))
+    replace_item_operation = ("replace", ("replace_item", {"id": "replace_item", "message": "item was replaced"}))
+    replace_item_if_match_operation = ("replace",
+                                       ("replace_item", {"id": "replace_item", "message": "item was replaced"}),
+                                       {"if_match_etag": container.client_connection.last_response_headers.get("etag")})
+    replace_item_if_none_match_operation = ("replace",
+                                           ("replace_item", {"id": "replace_item", "message": "item was replaced"}),
+                                           {"if_none_match_etag":
+                                                container.client_connection.last_response_headers.get("etag")})
+
+    # Put our operations into a list
+    batch_operations = [
+        create_item_operation,
+        upsert_item_operation,
+        read_item_operation,
+        delete_item_operation,
+        replace_item_operation,
+        replace_item_if_match_operation,
+        replace_item_if_none_match_operation]
+
+    # Run that list of operations
+    batch_results = container.execute_item_batch(batch_operations=batch_operations, partition_key="Account1")
```

Comment:
```suggestion
    try:
        batch_results = container.execute_item_batch(batch_operations=batch_operations, partition_key="Account1")
    except CosmosBatchOperationError as error:
        # Now what should i do with this information?
        # Are there particular status codes that I should be retring on? What are the common scenarios here?
        problem_item = batch_operations[error.error_index]
```

Reply:
I'll hold off on this until we see what we're doing with the errors